src/Symfony/Component/Semaphore/Tests/SemaphoreTest.php

```diff
@@ -252,4 +252,21 @@ public function testExpiration()
         $semaphore = new Semaphore($key, $store);
         $this->assertTrue($semaphore->isExpired());
     }
+
```

Comment:
Add `@group time-sensitive` so we dont actually wait for a second. That would slow down the tests.

Reply:
good idea, much cleaner than my 100ms ttl